src/attributes.rst

```diff
@@ -866,61 +866,24 @@ Attribute ``doc``
 .. syntax::
 
    DocContent ::=
```

Comment:
One possibility might be

DocContent ::=
    $$doc$$ DocInput

DocInput ::=
    $$=$$ MacroInvocation
  | $$=$$ StringLiteral
  | $$($$ TokenTree* $$)$$